docs/versions/unversioned/sdk/google.md

```diff
@@ -2,165 +2,177 @@
-    8.  Press **Save**.
--   **Get an OAuth client ID for your app**
-    1.  Build a standalone app and download the apk, or find one that you have already built.
-    2.  Go to the [Google Developer Credentials](https://console.developers.google.com/apis/credentials).
-    3.  Click **Create credentials**, then **OAuth client ID**, then select the **Android** radio button.
-    4.  Run `expo fetch:android:hashes`.
-    5.  Take `Google Certificate Fingerprint` from previous step and insert it in the **Signing-certificate fingerprint** field.
-    6.  Add your `android.package` from `app.json` (eg: `ca.brentvatne.growlerprowler`) to the **Package name** field.
-    7.  Press **Create**.
--   **Add the configuration to your app**
-    1.  Build a standalone app and download the apk, or find one that you have already built.
-    2.  Go to the [Google Developer Credentials](https://console.developers.google.com/apis/credentials) and find your API key.
-    3.  Open `app.json` and add your **Google API Key** to `android.config.googleSignIn.apiKey`.
-    4.  Run `expo fetch:android:hashes`.
-    5.  Take `Google Certificate Hash` from the previous step to `app.json` under `android.config.googleSignIn.certificateHash`.
-    6.  When you use `Expo.Google.logInAsync(..)`, pass in the **OAuth client ID** as the `androidStandaloneAppClientId` option.
-    7.  Rebuild your standalone app.
-
-Note that if you've enabled Google Play's app signing service, you will need to grab their app signing certificate in production rather than the upload certificate returned by `expo fetch:android:hashes`. You can do this by grabbing the signature from Play Console -> Your App -> Release management -> App signing, and then going to the [API Dashboard](https://console.developers.google.com/apis/) -> Credentials and adding the signature to your existing credential.
-
-## Deploying to a standalone app on iOS
-
-If you want to use native sign in for a standalone app, you can follow these steps. These steps assume that you already have it working on the Expo client app.
-
-1.  Add a `bundleIdentifier` to your `app.json` if you don't already have one.
-2.  Open your browser to [Google Developer Credentials](https://console.developers.google.com/apis/credentials)
-3.  Click **Create credentials** and then **OAuth client ID**, then choose **iOS**.
-4.  Provide your `bundleIdentifier` in the **Bundle ID** field, then press **Create**.
-5.  Add the given **iOS URL scheme** to your `app.json` under `ios.config.googleSignIn.reservedClientId`.
-6.  Wherever you use `Expo.Google.logInAsync`, provide the **OAuth client ID** as the `iosStandaloneAppClientId` option.
-7.  Rebuild your standalone app.
+In the Expo client app, you can only use browser-based login (this works very well actually because it re-uses credentials saved in your system browser). If you build a standalone app, you can use the native login with the package `expo-google-sign-in`.
```

Comment:
```suggestion
In the ExpoClient app, you can only use browser-based login (this works very well actually because it re-uses credentials saved in your system browser). If you build a standalone app, you can use the native login with the package `expo-google-sign-in`.
```